tests/components/modbus/test_init.py

```diff
@@ -137,6 +143,26 @@ async def mock_modbus_with_pymodbus_fixture(hass, caplog, do_config, mock_pymodb
     return mock_pymodbus
 
 
+async def test_fixedRegList_validator() -> None:
+    """Test fixed temp registers validator."""
+
+    registers: list(int) = [30, 31, 32, 33, 34, 35, 36]
+
+    for value in (
+        15,
+        registers,
+    ):
+        assert isinstance(hvac_fixedsize_reglist_validator(value), list)
+
+    with contextlib.suppress(vol.Invalid):
```

Comment:
Use pytest.

Reply:
> Use pytest.

What do you mean? I used pytest, what is it wrong?

Reply:
Sorry, I misunderstood. fixed, now